django/contrib/admin/tests.py

```diff
@@ -14,11 +15,41 @@ def process_response(self, request, response):
         return response
 
 
+class AdminSeleniumMetaClass(type):
+    def __new__(cls, name, bases, attrs):
+        """
+        Dynamically inject new methods for running tests in different browsers
+        when multiple browser specs are provided (e.g. --selenium=firefox,chrome).
+        """
+        all_specs = os.environ.get('DJANGO_SELENIUM_SPECS', '').split(',')
+        if all_specs:
+            for key, value in attrs.items():
+                if isinstance(value, types.FunctionType) and key.startswith('test'):
+                    func = value
```

Comment:
I guess all "func" names could be called "method" since we're dealing with test methods, not functions.